charts/incubator/automatic-music-downloader/questions.yaml

```diff
@@ -0,0 +1,485 @@
+          schema:
+            type: boolean
+            default: true
+        - variable: LIST
+          label: "LIST"
+          description: "Both or missing or cutoff :: both = missing + cutoff :: missng = lidarr missing list :: cutoff = lidarr cutoff list"
+          schema:
+            type: string
+            default: "both"
+            enum:
+              - value: "both"
+                description: "missing + cutoff"
+              - value: "missing"
+                description: "lidarr missing list"
+              - value: "cutoff"
+                description: "lidarr cutoff list"
+        - variable: SearchType
+          label: "Search Type"
+          description: "both or artist or fuzzy :: both = artist + fuzzy searching :: artist = only artist searching :: fuzzy = only fuzzy searching (Various Artist is always fuzzy searched, regardless of setting)"
+          schema:
+            type: string
+            default: "both"
+            enum:
+              - value: "both"
+                description: "artist + fuzzy searching "
+              - value: "artist"
+                description: "artist = only artist searching"
+              - value: "fuzzy"
+                description: "only fuzzy searching (Various Artist is always fuzzy searched, regardless of setting)"
+        - variable: Concurrency
+          label: "Concurrency"
+          description: "Number of concurrent downloads"
+          schema:
+            type: int
+            default: 1
+        - variable: EMBEDDED_COVER_QUALITY
+          label: "Embedded Cover Quality"
+          description: "Controls the quality of the cover image compression in percentage, 100 = no compression"
+          schema:
+            type: int
+            default: 80
+        - variable: FORMAT
+          label: "Format"
+          description: "FLAC or MP3 or OPUS or AAC or ALAC"
+          schema:
+            type: string
+            default: "FLAC"
+            enum:
+              - value: "FLAC"
+                description: "FLAC is an audio coding format for lossless compression of digital audio"
+              - value: "MP3"
+                description: "MP3 (or mp3) as a file format commonly designates files containing an elementary stream of MPEG-1 Audio or MPEG-2 Audio encoded data"
+              - value: "OPUS"
+                description: "Opus is a lossy audio coding format developed by the Xiph.Org Foundation and standardized by the Internet Engineering Task Force"
+              - value: "AAC"
+                description: "Advanced Audio Coding (AAC) is an audio coding standard for lossy digital audio compression."
+              - value: "ALAC"
+                description: "The Apple Lossless Audio Codec (ALAC), also known as Apple Lossless, or Apple Lossless Encoder (ALE), is an audio coding format, and its reference audio codec implementation, developed by Apple Inc."
```

Comment:
Those descriptions might be a bit long for the dropdowns.
( The description is what the users see )
I'd prefer something shorter, as I'm not sure how SCALE's UI will handle those

Descriptions outside on enums are hidden under then `?`. So they can be long.

Reply:
2 words, maybe 3 max, yes.

Reply:
Not your fault though, I don't think "description" is the right word here, because it's not actually the description, it's the label >.<